srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=1
+archs="x86_64 x86_64-musl i686"
+wrksrc="ocs-url-release-${version}"
+build_style="qmake"
+configure_args="PREFIX='/usr'"
+make_build_args="INSTALL_ROOT=${DESTDIR} -j${npoc}"
+make_install_args="INSTALL_ROOT=${DESTDIR} -j${npoc}"
```

Comment:
Build style pass these already. For parallel build, set a variable in etc/conf, no need for`-j$(nproc)` ;)

Reply:
Understood! :) I added as it took 3 hours to build on my laptop without it, I'll remove it now

Reply:
Resolved!